opsdroid/helper.py

```diff
@@ -28,7 +37,20 @@ def del_rw(action, name, exc):
 
 
 def move_config_to_appdir(src, dst):
-    """Copy any .yaml extension in "src" to "dst" and remove from "src"."""
+    """ This method allows to make copy of files with .yaml extension
+        from  "src" folder to "dst" folder and remove the current file in "src".
+
+    Args:
+        src (str): path file.
+        dst (str): destination path
+    Raises:
```

Comment:
The logging isn't really something being raised although I do understand why you would choose this one, perhaps this should be replaced with "Attributes" as you can see in the example here:

```python
class ExampleError(Exception):
    """Exceptions are documented in the same way as classes.

    The __init__ method may be documented in either the class level
    docstring, or as a docstring on the __init__ method itself.

    Either form is acceptable, but the two should not be mixed. Choose one
    convention to document the __init__ method and be consistent with it.

    Note:
        Do not include the `self` parameter in the ``Args`` section.

    Args:
        msg (str): Human readable string describing the exception.
        code (:obj:`int`, optional): Error code.

    Attributes:
        msg (str): Human readable string describing the exception.
        code (int): Exception error code.

    """
```

If possible I think mentioning that the files in the 'src' will be deleted once the files are copied and a check is made to see if both files are indeed the same (this refers to the code on line 46-47).

You can even rename

Reply:
1) the function take 2 parameters   1- source path 2- destination path. like you can see below: 
```
def function_with_types_in_docstring(param1, param2):
    """Example function with types documented in the docstring.

    `PEP 484`_ type annotations are supported. If attribute, parameter, and
    return types are annotated according to `PEP 484`_, they do not need to be
    included in the docstring:

    Args:
        param1 (int): The first parameter.
        param2 (str): The second parameter.

    Returns:
        bool: The return value. True for success, False otherwise.

    .. _PEP 484:
        https://www.python.org/dev/peps/pep-0484/

    """
```